lib/mongo/insertOne.js

```diff
@@ -0,0 +1,31 @@
+const Streamz = require('streamz');
+const Promise = require('bluebird');
+const util = require('util');
+
+function insertOne(_c,collection,options) {
+  if (!(this instanceof Streamz))
+    return new insertOne(_c,collection);
+
+  if (isNaN(_c)) {
+    options = collection;
+    collection = _c;
+    _c = undefined;
+  }
+
+  Streamz.call(this, _c, null, options);
+  this.collection = Promise.resolve(collection);
+  this.options = options || {};
+}
+
+util.inherits(insertOne,Streamz);
+
+insertOne.prototype._fn = function(d) {
+  return this.collection
+    .then(collection =>collection.insertOne(d,this.options))
```

Comment:
Nice work on this PR! Would this work followed by an etl.collect(10)? I am getting doc has to be object error, which I assume means that due to a collect this is passing an array of docs. I think the docs in this PR should reflect this, remove the collect(10) in the example maybe?